catalog-rest-service/src/main/resources/json/schema/tests/test.json

```diff
@@ -0,0 +1,95 @@
+{
+  "$id": "https://open-metadata.org/schema/tests/test.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Test",
+  "description": "Test is a test definition to capture data quality tests against tables and columns.",
+  "type": "object",
+  "javaType": "org.openmetadata.catalog.tests.Test",
+  "javaInterfaces": ["org.openmetadata.catalog.EntityInterface"],
+  "definitions": {
+    "testCaseParameterValue": {
+      "type": "object",
+      "javaType": "org.openmetadata.catalog.test.TestCaseParameterValue",
+      "description": "This schema defines the parameter values that can be passed for a Test Case.",
+      "properties": {
+        "name": {
+          "description": "name of the parameter. Must match the parameter names in testCaseParameterDefinition",
+          "type": "string"
+        },
+        "value": {
+          "description": "value to be passed for the Parameters. These are input from Users. We capture this in in string and convert during the runtime.",
+          "type": "string"
```

Comment:
Should we accept arrays as well? i am think of test cases such as "values to be in set", wdyt?